### PR TITLE
FE for Detailed post. Included comments feature (new, likes show)

### PR DESCRIPTION
**Pull request by nibab-boo**
Sorry, This branch may have included more than needed.

- [x] FE for Post show
- [x] Show Comments
- [x] New Comments
- [x] Like/Unlike Comments ( Have problem with BE related to Unlike Comments)

### CODE REVIEW

File: frontend-next/api/posts/comments/actions.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+export function onLikedComment(postId: string, commentId: string) {
+  return new Promise((resolve: (response: any) => void, reject) =>
+    put(`/api/v2/posts/${postId}/comments/${commentId}/like`, {})
+      .then(response => {
+        resolve(response);
+      }).catch(error => {
+        reject(error);
+      })
+  );
+};
```
Comment (lalomartins) [inline frontend-next/api/posts/comments/actions.ts:25]: This Promise constructor is redundant.

```suggestion
export function onLikedComment(postId: string, commentId: string) {
  return put(`/api/v2/posts/${postId}/comments/${commentId}/like`, {});
};
```

File: frontend-next/api/posts/actions.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-import { resolve } from 'path';
 import { post } from 'utils/request';
- 
-export function createPost(postBody: {}, currentIdentity: string) {
+import { CreatePostBodyType } from '@models/post';
+
+type CreatePostType = (postBody: CreatePostBodyType, currentIdentity: string)  => Promise<any>
+
+export const createPost: CreatePostType = (postBody: CreatePostBodyType, currentIdentity: string) => {
   return new Promise((resolve: (response: any) => void, reject) =>
```
Comment (lalomartins) [inline frontend-next/api/posts/actions.ts:7]: Promise constructor is redundant.

**Review by lalomartins — CHANGES_REQUESTED** [review]
mostly looks good, just clean up the promises 😸

File: frontend-next/api/posts/actions.ts (hunk 2 above)
Comment (nibab-boo) [reply frontend-next/api/posts/actions.ts]: @lalomartins Done.

**Review by lalomartins — APPROVED** [review]

**Pull request merged**